tiledb/ml/models/_cloud_utils.py

```diff
@@ -29,3 +29,14 @@ def update_file_properties(uri: str, file_properties: Dict[str, str]) -> None:
         file_type=FILETYPE_ML_MODEL,
         file_properties=file_properties,
     )
+
+
+def get_cloud_uri(uri: str, namespace: Optional[str]) -> str:
+    s3_prefix = get_s3_prefix(namespace=namespace)
+    if s3_prefix is None:
+        raise ValueError(
+            f"You must set the default s3 prefix path for ML models in "
+            f"{namespace} profile settings on TileDB-Cloud"
+        )
+
+    return "tiledb://{}/{}".format(namespace, os.path.join(s3_prefix, uri))
```

Comment:
You can use `fstring` in this return same as above. It is slightly more comprehensive